guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -95,21 +92,13 @@ ifdef::satellite[]
 endif::[]
 
 ifdef::satellite[]
-* For the following {RHEL} 7 repositories required to build the Discovery image, change the download policy to *Immediate*.
+* For the following {EL} 8 repositories required to build the Discovery image, change the download policy to *Immediate*.
 This is required because {Project} downloads all packages only during synchronization of repositories with the immediate download policy.
-+
-** The latest *{RHEL} 7 Server (Kickstart)* repository.
-For example, *{RHEL} 7 Server Kickstart x86_64 7.7*.
-** *Red Hat Satellite Capsule {ProductVersion} for RHEL 7 Server RPMs x86_64*.
```

Comment:
This should be replaced with the RHEL 8 equivalents.

Reply:
Addressed but it may require a review.